python-connectors/pi-system_event-frames-search/connector.py

```diff
@@ -63,8 +65,10 @@ def generate_rows(self, dataset_schema=None, dataset_partitioning=None,
                 **self.config
             )
             headers = self.client.get_requests_headers()
+            # pagination = Pagination(self.client.get, self.client.get, self.database_endpoint + "/eventframes", headers=headers, params=params, error_source="generate_rows")
```

Comment:
same as above

Reply:
Removed in [0e8d31f](https://github.com/dataiku/dss-plugin-osisoft/pull/17/commits/0e8d31feacfaa01e6ac699ff34e6a6d07754c94f)